build/bundle-konflux.Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 as image-replacer
 COPY bundle/manifests /manifests
-RUN sed -i "s|REPLACE_IMAGE|quay.io/redhat-user-workloads/windows-machine-conf-tenant/windows-machine-config-operator/windows-machine-config-operator-release-4-17@sha256:c7325b1be0d2ec2849d337983f093748c31c058f80f55b5e8a5244fd81d16c0e|g" /manifests/windows-machine-config-operator.clusterserviceversion.yaml
+RUN sed -i "s|REPLACE_IMAGE|registry.redhat.io/openshift4-wincw/windows-machine-config-rhel9-operator:10.17.1|g" /manifests/windows-machine-config-operator.clusterserviceversion.yaml
```

Comment:
What about including the `bundle-konflux.Dockerfile` file in the list of `skip-if-changed`  so that  CI does not block this.

Reply:
Good idea